i18n/pt_BR.csv

```diff
@@ -1,34 +1,115 @@
+"Enviamos una copia al correo electrónico:","Enviamos uma cópia para o e-mail:"
+"Print Baloto","Imprimir Baloto"
+"Print OXXO","Print OXXO"
+"Print SPEI","Print SPEI"
+"View CIP","Visualizar CIP"
+"Buy with 1-Click","Comprar com 1-Click"
+"Choose your shipping address","Selecione seu endereço de envio"
+"Choose the shipping method","Selecione o método de envio"
+"Select a previously saved credit card to pay your order","Selecione um cartão de crédito salvo previamente para pagar seu pedido"
+"Document number","Número do documento"
+"Place Order","Finalizar Pedido"
+"Credit Card Information","Dados do Cartão de Crédito"
+"Use a previously saved credit card","Use um cartão de crédito salvo previamente"
+Installments,Parcelas
+"Credit Card Holder Name","Nome do Portador do Cartão"
+"Credit Card Number","Número"
+"Expiration Date","Data de Validade"
+"Card Verification Number","CVV"
+"What is this?","O que é isso?"
+"Save credit card data to your account?","Salvar dados do cartão na sua conta?"
+"Pay with","Pagar com"
+"Continue to payment","Continuar para pagamento"
+TEF,TEF
+"Choose your bank","Selecione o seu banco"
+Bank,Banco
+"Use previously saved credit card","Use um cartão de crédito salvo previamente"
+"Document Type","Tipo de Documento"
+"Document Number","Número do documento"
+"Select the method","Selecione o método"
+"Credit/Debit Card Information","Informações do cartão de crédito/débito"
+"Please, select all required product options","Por favor, selecione todas as opções obrigatórias do produto"
+"An error ocurred when trying place your order.","Ocorreu um erro ao finalizar seu pedido."
+"x of ","x de "
+" (with interest)"," (com juros)"
+"Use a different credit card","Usar outro cartão"
+"Total amount in local currency:","Valor total em moeda local:"
+"Token generation error. Please contact support.","Erro ao gerar um token. Por favor, contate o suporte."
+"Invalid Document Length","Tamanho do número de documento inválido"
+"Invalid Credit Card Data","Dados do cartão inválidos"
+"Total amount in local currency with IOF (0.38%):","Valor total em moeda local com IOF (0.38%):"
+"Invalid Document","Documento inválido"
+Cash,Dinheiro
+On-line,On-line
+"EBANX Pay","EBANX Pay"
+Active,Ativo
+"Sandbox Mode","Modo Sandbox"
+"Integration Key","Integration Key"
+"Public Integration Key","Public Integration Key"
+Debug,Debug
+"Show total local amount","Exibir total em moeda local"
+"Apply IOF to local amount","Aplicar IOF em moeda local"
+"Brazil payments","Pagamentos no Brasil"
+"Credit Card configuration","Configurações de Cartão de Crédito"
+"Save credit card","Salvar cartão de crédito"
+"1-Click payment","Pagamento com 1-Click"
+Auto-Capture,Auto-Capture
+"Max installments number","Número máximo de parcelas"
+"Min installment value","Valor mínimo de parcela"
+"Enable interest rates","Habilitar taxas para parcelado"
+"Interest rates","Taxa de parcelado"
```

Comment:
Shouldn't be `Juros` or `Taxa de Juros`? `Taxa de parcelado` seems wrong...

Reply:
Yep... seems better. Just to be in the context: This is a text that shows up when the customer is configuring his checkout, so it's related  to allow the customer to apply a fee in Transactions with Interest. That's why I translated to "Taxa de parcelado". But in any case, "Taxa de Juros" sounds better.